### PR TITLE
Data Connector Description Update

### PR DESCRIPTION
**Pull request by v-prasadboke**
Required items, please complete
   
   Change(s):
   Data Connector Description Update for
   - Squid Proxy
   - Symantec Endpoint Protection
   - Symantec VIP

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @v-rbajaj, please provide your feedback. Thanks

**Comment by v-rbajaj** [comment]
Hi @v-prasadboke, for squid proxy please update the zip file, the zip 2.0.1 includes 2.0.0 zip, please remove 2.0.0 zip.

**Comment by v-prasadboke** [comment]
> Hi @v-prasadboke, for squid proxy please update the zip file, the zip 2.0.1 includes 2.0.0 zip, please remove 2.0.0 zip.
> 
> Also please update squid proxy create ui as per below image image

resolved @v-rbajaj

File: Solutions/VMware vCenter/Data/Solution_vcenter.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   ], 
   "Metadata": "SolutionMetadata.json",
   "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\VMware vCenter",
-  "Version": "2.0.0",
+  "Version": "2.0.2",
```
Comment (v-rbajaj) [inline Solutions/VMware vCenter/Data/Solution_vcenter.json:21]: Can you please confirm the version ? Shouldn't it be 2.0.1?

File: Solutions/Symantec Endpoint Protection/Data/Solution_Symantec.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "Parsers/SymantecEndpointProtection.txt"
     ],
     "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\Symantec Endpoint Protection",
-	"Version": "2.0.2",
+	"Version": "2.0.4",
```
Comment (v-rbajaj) [inline Solutions/Symantec Endpoint Protection/Data/Solution_Symantec.json:20]: Can you please confirm the version, it should be 2.0.3, right?

**Comment by v-rbajaj** [comment]
> > Hi @v-prasadboke, for squid proxy please update the zip file, the zip 2.0.1 includes 2.0.0 zip, please remove 2.0.0 zip.
> > Also please update squid proxy create ui as per below image image
> 
> resolved @v-rbajaj

Please update the `description` in the `Solutions/SquidProxy/Data/Solution_SquidProxytemplateSpec.json`

**Comment by v-rbajaj** [comment]
@v-prasadboke, please discard changes in create ui for Symantec VIP, Symantec EndPointProtection and VMware vCenter in data connector and parser description

File: Solutions/SquidProxy/Package/mainTemplate.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
     "_dataConnectorContentId1": "[variables('dataConnectorContentId1')]",
     "dataConnectorId1": "[extensionResourceId(resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspace')), 'Microsoft.SecurityInsights/dataConnectors', variables('_dataConnectorContentId1'))]",
     "_dataConnectorId1": "[variables('dataConnectorId1')]",
-    "dataConnectorTemplateSpecName1": "[concat(parameters('workspace'),'-DataConnector-',variables('_dataConnectorContentId1'))]",
+    "dataConnectorTemplateSpecName1": "[concat(parameters('workspace'),'-dc-',uniquestring(variables('_dataConnectorContentId1')))]",
```
Comment (v-rbajaj) [inline Solutions/SquidProxy/Package/mainTemplate.json:44]: Hi @v-prasadboke, please revert this change!

File: Solutions/Symantec Endpoint Protection/Package/createUiDefinition.json [hunk 4]
```diff
@@ -67,7 +67,7 @@
             "name": "dataconnectors-parser-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "The Symantec Endpoint Protection Solution installs a parser that transforms the ingested data into Microsoft Sentinel normalized format. The normalized format enables better correlation of different types of data from different data sources to drive end-to-end outcomes seamlessly in security monitoring, hunting, incident investigation and response scenarios in Microsoft Sentinel."
+              "text": "The Symantec Endpoint Proection Solution installs a parser that transforms the ingested data into Microsoft Sentinel normalized format. The normalized format enables better correlation of different types of data from different data sources to drive end-to-end outcomes seamlessly in security monitoring, hunting, incident investigation and response scenarios in Microsoft Sentinel."
```
Comment (v-rbajaj) [inline Solutions/Symantec Endpoint Protection/Package/createUiDefinition.json:70]: Please correct the spelling of "Protection"

**Review by v-rbajaj — APPROVED** [review]
Looks fine

**Review by v-dvedak — APPROVED** [review]
